index.html

```diff
@@ -289,21 +342,23 @@ <h3><a class="about-connect" href="#">LET'S CONNECT</a></h3>
     </section>
 
     <!-- Contact-Form -->
-    <section class="contact-form" id="Contact">
-      <h2 class="contact-title" >Contact me</h2>
-      <p class="contact-info" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+    <section class="contact-form form-group" id="contact">
+      <h2 class="contact-title">Contact me</h2>
+      <p class="contact-info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
       
       <fieldset class="fieldset">
-        <form action="https://formspree.io/f/xaykrzrn" method="post">
+        <form action="https://formspree.io/f/xaykrzrn" method="post" id="contact-form">
         <ul>
           <li><input type="text" class="form-input" name="name" placeholder="Your Name" maxlength="30" required></li>
-          <li><input type="email" class="form-input" name="email" placeholder="Your Email" required></li>
+          <li><input type="email" id="email" class="form-input" name="email" placeholder="Your Email" required></li>
+          <p class="error" id="error"></p>
           <li><textarea class="form-input" id="text-area" name="text-area" placeholder="Write your message here" maxlength="500" required></textarea></li>
           <button class="contact-button" type="submit" name="button">Get In Touch</button>
         </ul>
```

Comment:
- [ ] This activity requires that the error message is displayed around the submit button. Kindly ensure to improve upon the same. 
![image](https://user-images.githubusercontent.com/12745474/211143336-2d958f41-6a6a-43e1-bdca-4dfdd6fa5e19.png)

- [ ] OPTIONAL
Your email input and message input fields appear joined on desktop screens. Could you make sure to fix it by adding some spacing between the two fields?

![image](https://user-images.githubusercontent.com/12745474/211143469-0e7dc03b-1980-43e3-9511-b4b631f5fd2e.png)